platforms/corda-ent/configuration/deploy-network.yaml

```diff
@@ -122,10 +122,10 @@
     when: org.type == 'node'
 
   # ----------------------------------------------------------------------   
-  # Create value file for node registration
-  - name: Create value file for node registration
+  # Create value file for node
+  - name: Create value file for node
     include_role:
-      name: setup/node_registration
+      name: setup/node
```

Comment:
move the tls for node, bridge and float setup inside the node setup role

Reply:
done